projects/sonar/src/app/record/document/file/file.component.html

```diff
@@ -40,7 +40,7 @@
         </ng-container>
         <ng-template #downloadLink>
           <ng-container *ngIf="showDownload && file.links.download; else noLink">
-            <a href="{{ file.links.download }}?download" *ngIf="showDownload && file.links.download">
```

Comment:
Why did you remove `ngIf` the property `showDownload` still used in the component?

Reply:
Because it is already in the line above:
<ng-container *ngIf="showDownload && file.links.download; else noLink">